frontend/www/js/omegaup/components/user/PreferencesEdit.vue

```diff
@@ -36,6 +36,48 @@
         </option>
       </select>
     </div>
+    <div class="form-group">
+      <label>{{ T.userObjectivesModalDescriptionUsage }}</label>
+      <select
+        v-model="firstObjective"
+        data-firstObjective
+        class="custom-select"
+      >
+        <option :value="ObjectivesAnswers.Learning">
+          {{ T.userObjectivesModalAnswerLearning }}
+        </option>
+        <option :value="ObjectivesAnswers.Teaching">
+          {{ T.userObjectivesModalAnswerTeaching }}
+        </option>
+        <option :value="ObjectivesAnswers.LearningAndTeaching">
+          {{ T.userObjectivesModalAnswerLearningAndTeaching }}
+        </option>
+        <option :value="ObjectivesAnswers.None">
+          {{ T.userObjectivesModalAnswerNone }}
+        </option>
+      </select>
+    </div>
+    <div class="form-group">
+      <label>{{ T.userObjectivesModalDescriptionUsage }}</label>
+      <select
+        v-model="secondObjective"
+        data-secondObjective
```

Comment:
```suggestion
        data-second-objective
```

acá igual, quizás sería mejor `data-scholar-competitive-objective`.